includes/multi-currency/class-compatibility.php

```diff
@@ -0,0 +1,127 @@
+<?php
+/**
+ * Class Compatibility
+ *
+ * @package WooCommerce\Payments\Compatibility
+ */
+
+namespace WCPay\Multi_Currency;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class that controls Multi Currency Compatibility.
+ */
+class Compatibility {
+	/**
+	 * Multi-Currency instance.
+	 *
+	 * @var Multi_Currency
+	 */
+	protected $multi_currency;
```

Comment:
It doesn't look like we use the instance of `$multi_currency` in this class, so perhaps we don't need to pass this in or even have it a property within this class?

This may make things easier when it comes to testing as it simplifies the class hierarcy.

Reply:
Good call on this, it has been removed 😁